apps/site/config/test.exs

```diff
@@ -14,7 +14,7 @@ config :site, :secure_pipeline,
   ]
 
 # Print only warnings and errors during test
-config :logger, level: :warn
+config :logger, level: :notice
```

Comment:
If this change was intentional the comment above should be updated to reflect that